From a68c07024e45365ab04fdab86a429b1dbc595a64 Mon Sep 17 00:00:00 2001
From: Paulo Zanoni <pzanoni@mandriva.com>
Date: Tue, 1 Mar 2011 10:38:01 -0300
Subject: [PATCH 907/908] Add "-nr" argument for backwards compatibility

We used to have patch implementing this argument. The patch was integrated
upstream but as "-background none" instead of "-nr". Let's keep the "-nr"
option while we still have "-nr" users.
---
diff -Naur xorg-server-1.17.0/os/utils.c xorg-server-1.17.0.tpg/os/utils.c
--- xorg-server-1.17.0/os/utils.c	2014-11-17 05:51:18.000000000 +0000
+++ xorg-server-1.17.0.tpg/os/utils.c	2015-02-05 14:08:53.791982571 +0000
@@ -560,6 +560,7 @@
     ErrorF("-listen string         listen on protocol\n");
     ErrorF("-noreset               don't reset after last client exists\n");
     ErrorF("-background [none]     create root window with no background\n");
+    ErrorF("-nr                    same as '-background none'\n");
     ErrorF("-reset                 reset after last client exists\n");
     ErrorF("-p #                   screen-saver pattern duration (minutes)\n");
     ErrorF("-pn                    accept failure to listen on all ports\n");
@@ -884,6 +885,9 @@
         else if (strcmp(argv[i], "-reset") == 0) {
             dispatchExceptionAtReset = DE_RESET;
         }
+        else if ( strcmp( argv[i], "-nr") == 0) {
+    	    bgNoneRoot = TRUE;
+    	}
         else if (strcmp(argv[i], "-p") == 0) {
             if (++i < argc)
                 defaultScreenSaverInterval = ((CARD32) atoi(argv[i])) *
-- 
1.7.6
